scene/3d/path_3d.cpp

```diff
@@ -230,11 +233,11 @@ void PathFollow3D::_notification(int p_what) {
 	}
 }
 
-void PathFollow3D::set_cubic_interpolation(bool p_enable) {
-	cubic = p_enable;
+void PathFollow3D::set_cubic_interpolation_enabled(bool p_enabled) {
+	cubic = p_enabled;
 }
 
-bool PathFollow3D::get_cubic_interpolation() const {
+bool PathFollow3D::is_cubic_interpolation_enabled() const {
```

Comment:
These changes seem unrelated to adding a new model front mode to `look_at`. I'm not sure why not split these changes into a separate PR.

Reply:
I talked to @akien-mga earlier and got his agreement that these method names are bad, but he seems to have a terrible aversion to changing method names in 4.x, so I decided to change only the cpp function names. same as https://github.com/godotengine/godot/pull/74190#issuecomment-1546352973. Binding method names have not been changed. If he does not like it, I will split it.